docs/meshstack.aks-platform-quickstart-guide.md

```diff
@@ -0,0 +1,223 @@
+
+# Prerequisites
+
+Before starting, ensure you have the following:
+
+- An AKS Cluster that is publicly accessible via the internet
+- A place to run Terraform. For example, your local machine, or the Azure cloud shell.
+
+# Preparing Infrastructure in Azure
+
+To integrate your AKS cluster into meshStack properly, we require a few things to be in place in AKS & Azure for a successful integration.
+
+In this guide, we will use our Terraform module to set these things up. You can also do this manually, but we highly recommend doing this with Terraform, and for this guide we assume you did.
+
+Amongst other things, the Terraform module will create a service user with the right permissions in the Kubernetes cluster and create a service principal in Azure to interact with your Entra tenant.
+
+Click [here](https://github.com/meshcloud/terraform-aks-meshplatform) and follow along the guide in the README to use the Terraform module.
+
+Make sure you execute the last step of exporting the outputs and save these somewhere. We need them later in this guide.
+
+# Creating the Platform
+
+1. **Open the platform list in the service management area**
+
+   When entering your workspace, click the blue “Go to Services Management” button at the top-right if you haven’t already. Then open the menu on the left-hand side and click on “Platform → Platforms”.
+
+2. **Create a new platform**
+
+   In the list, click on the button “+ Create new platform”. A wizard will open where we will have to enter some information.
+
+3. **Select platform type**
+
+   Under “Platform Type” pick “Azure Kubernetes Service”.
+
+4. **Enter name, identifier & description**
+
+   Enter a fitting name, for example “Azure Kubernetes Service”. You can pick a more specific name if you have multiple clusters for example.
+
+   For the identifier pick something similar to the name, for example `aks`.
+
+   For the description you can write a little bit of additional information about your cluster that people can see in the marketplace. You can also leave this empty for now.
+
+5. **Enter Cloud Platform Web Console URL**
+
+   Enter a URL here that users can use to open the web console of your AKS cluster. You can enter the API Server Address that can be found in Azure Portal when opening up your AKS cluster.
+
+   The support & documentation fields below can be left empty.
+
+6. **Create a new location**
+
+   Locations are a grouping mechanism for multiple platforms in meshStack. For now you can create a new location and give it the name and identifier “AKS”. After that, click “Next”.
+
+7. **Pricing**
+
+   You will be prompted to enter a price for your new platform. You can skip this for now.
+
+
+🎉 The platform is created in meshStack! But it won’t be usable yet. We will have to properly integrate it now
+
+
+# Configuring Integration
+
+1. **Open the Configuration**
```

Comment:
Please fix capitalization. The list above is all small besides the first word the list below uses capitalization used in headings